corehq/apps/hqwebapp/views.py

```diff
@@ -249,66 +249,38 @@ def server_up(req):
     Hit serverup.txt?only={check_name} to only check a specific service
     Hit serverup.txt?{check_name} to include a non-default check (currently only ``heartbeat``)
     """
-
-    checkers = {
```

Comment:
this didn't include all the checks. Missing ones were, `kafka`, `touchforms`, `blobdb`, `rabbitmq`